proto/pulumi/analyzer.proto

```diff
@@ -65,8 +65,11 @@ message AnalyzerStackConfigureRequest {
     string organization = 3;
     // True if this is a preview/dry run.
     bool dry_run = 4;
+
+    reserved 5; // This was used to send stack config as property values.
+
     // The configuration of the stack being analyzed.
-    map<string, string> config = 5;
+    map<string, string> config = 7;
```

Comment:
Nit: Would it be worth keeping the fields in the same order as the index? I personally find it easier to reason about when they are in order, when adding new fields in the future.